Chatto/Source/ChatController/BaseChatViewController.swift

```diff
@@ -167,7 +176,13 @@ open class BaseChatViewController: UIViewController, UICollectionViewDataSource,
             sSelf.isAdjustingInputContainer = false
         }
         self.keyboardTracker = KeyboardTracker(viewController: self, inputContainer: self.inputContainer, layoutBlock: layoutBlock, notificationCenter: self.notificationCenter)
-        (self.view as? BaseChatViewControllerView)?.bmaInputAccessoryView = self.keyboardTracker?.trackingView
+
+        // Compiler can't understand this cast inline. 
```

Comment:
This is because protocol is not restricted for classes only. Value types can implement it as well. Compiler doesn't allow to modify value type declared with `let` keyword. Can you fix it?